pandas/tests/indexes/test_datetimelike.py

```diff
@@ -721,6 +721,17 @@ def test_fillna_datetime64(self):
                            dtype=object)
             self.assert_index_equal(idx.fillna('x'), exp)
 
+    def test_contains(self):
```

Comment:
can u also add tests for ``TimedeltaIndex`` and ``PeriodIndex``? (It looks work)